src/events/processSignatureRequests/index.js

```diff
@@ -35,12 +35,17 @@ function processSignatureRequestsBuilder(config) {
       validatorContract = new web3Home.eth.Contract(bridgeValidatorsABI, validatorContractAddress)
     }
 
+    rootLogger.debug(`Processing ${signatureRequests.length} SignatureRequest events`)
     const callbacks = signatureRequests.map(signatureRequest =>
       limit(async () => {
         const { recipient, value } = signatureRequest.returnValues
 
+        const logger = rootLogger.child({
+          eventTransactionHash: signatureRequest.transactionHash
+        })
+
         logger.info(
-          { eventTransactionHash: signatureRequest.transactionHash, sender: recipient, value },
+          { sender: recipient, value },
           `Processing signatureRequest ${signatureRequest.transactionHash}`
         )
 
```

Comment:
consider to add a log after `sign` in the code below.

Reply:
The `createMessage` function is synchronous. We could add a log here (probably at `trace` level) but it's not something that should get stuck.